docker-compose.yaml

```diff
@@ -8,7 +8,7 @@ services:
             POSTGRES_USER: nango
             POSTGRES_DB: nango
         ports:
-            - '${NANGO_DB_PORT:-5432}:5432'
+            - '${NANGO_DB_PORT:-5436}:5432'
```

Comment:
Thanks for the contribution, can you remove this change updating the default port?